pkg/reconciler/apis/permissionclaimlabel/permissionclaimlabel_resource_reconcile.go

```diff
@@ -44,19 +44,14 @@ func (c *resourceController) reconcile(ctx context.Context, obj *unstructured.Un
 		return fmt.Errorf("error calculating permission claim labels for GVR %q %s/%s: %w", gvr, obj.GetNamespace(), obj.GetName(), err)
 	}
 
-	labels := obj.GetLabels()
-	if labels == nil {
-		labels = make(map[string]string)
-	}
```

Comment:
background for this commit: it is not good to depend on nil-vs-empty behaviour of `LabelsFor`.